### PR TITLE
Design changes

### PR DESCRIPTION
**Pull request by juliareichel**
-Improving CSS of the map state

### CODE REVIEW

File: src/main/webapp/creatives/maps/simid_map_creative.js [hunk 1]
```diff
@@ -289,20 +293,20 @@ export default class SimidMapCreative extends BaseSimidCreative {
...
+    const travelChoicesContainer = document.getElementById("adContainer")
     const travelMethod = document.createElement('select');
-    travelMethod.id = "travel_method";
+    travelMethod.id = "travelMethod";
     TRANSPORT_METHODS.forEach((transportType) =>{
       const newOption = this.createTravelOption_(transportType);
       travelMethod.add(newOption);
     });
-    travelMethod.classList.add("travel_method");
+    // travelMethod.classList.add("travel_method");
```
Comment (kristen-m) [inline src/main/webapp/creatives/maps/simid_map_creative.js:309]: we can get rid of this line!

Comment (juliareichel) [reply src/main/webapp/creatives/maps/simid_map_creative.js:309]: great, just wasn't sure why you had it initially, so was going to ask you

File: src/main/webapp/creatives/maps/simid_map_creative.js [hunk 2]
```diff
@@ -289,20 +293,19 @@ export default class SimidMapCreative extends BaseSimidCreative {
     this.calculateRoute_();
   }
 
-  /**
+   /**
```
Comment (Kaleab470) [inline src/main/webapp/creatives/maps/simid_map_creative.js:296]: stars should be aligned here

File: src/main/webapp/creatives/maps/simid_map.css [hunk 3]
```diff
@@ -59,12 +67,31 @@ html, body {
...
-    position: relative;
-    top: 170px;
+    top: 180px;
     -ms-transform: translateY(-50%);
     transform: translateY(-50%);
-    overflow: inherit !important;
+    zoom: 1.15;
+}
+
+.gm-style-mtc {
```
Comment (damargulis) [inline src/main/webapp/creatives/maps/simid_map.css:79]: just curious -- are these documented in the api and they say to style the map this way, or did you just inspect to see what the class values were?

File: src/main/webapp/creatives/maps/simid_map_creative.js (hunk 2 above)
Comment (juliareichel) [reply src/main/webapp/creatives/maps/simid_map_creative.js:296]: fixed that

**Pull request merged**